src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialog.fxml

```diff
@@ -62,6 +63,8 @@
                         <Tooltip text="%Searches the selected directory for unlinked files."/>
                     </tooltip>
                 </Button>
+                <CheckBox fx:id="storePreferencesCheck" GridPane.rowIndex="4"
+                        text="%Save selected options for later usage"/>
```

Comment:
Just always do that. No need to explicitly do that.

Reply:
Removed checkbox from dialog and all related to this checkbox stuff